src/Native/Runtime/windows/PalRedhawkMinWin.cpp

```diff
@@ -58,6 +61,28 @@ bool InitializeSystemInfo()
 
 extern bool PalQueryProcessorTopology();
 
+void __stdcall ThreadShutdown();
+
+// Callback that is called when a fiber is detached from a thread
```

Comment:
nit: to me, this line of the comment seems rather redundant on a method called `FiberDetachCallback`. Can you either elaborate some more on what's going on (i.e. how this is just the mechanism we use to perform thread detach outside of the loader lock) or just eliminate it altogether.